fio/fio2gnuplot.go

```diff
@@ -0,0 +1,60 @@
+package fio
+
+import (
+	"path/filepath"
+
+	"github.com/giantswarm/io-benchmarks/exec"
+
+	"github.com/juju/errgo"
+)
+
+type Fio2GNUPlotConfiguration struct {
+	LogsDirectory string
+}
+
+type Fio2GNUPlotRunner struct {
+	conf Fio2GNUPlotConfiguration
+}
+
+func NewFio2GNUPlotRunner(c Fio2GNUPlotConfiguration) (Fio2GNUPlotRunner, error) {
+	var err error
+
+	if !fio2GNUPlotExists() {
+		return Fio2GNUPlotRunner{}, errgo.Newf("Cannot locate fio2gnuplot. Looks like it is not installed on your system.")
+	}
+
+	if c.LogsDirectory, err = filepath.Abs(c.LogsDirectory); err != nil {
+		return Fio2GNUPlotRunner{}, errgo.Mask(err)
+	}
+
+	return Fio2GNUPlotRunner{
+		conf: c,
+	}, nil
+}
+
+func (r Fio2GNUPlotRunner) RunPlots() error {
+	patterns := []string{
+		"*_bw.*.log",
+		"*_iops.*.log",
+		"*_lat.*.log",
+		"*_clat.*.log",
```

Comment:
They are based on the logs that `fio` generates. I currently have no idea to make them more configurable. I could also make them their own type that is compatible to flags. All sounds very much like overblown.

Any ideas?

Sidenote: `fio2gnuplot` does not complain when I does not find any files matching to those patterns.